runbooks/update_appinspect_cli_action.md

```diff
@@ -0,0 +1,36 @@
+# Runbook to update splunk/appinspect-cli-action
+
+Once Splunk AppInspect team releases AppInspect CLI - we need to make sure that everyone runs the latest version.
+
+## Steps
+
+### Merge splunk/appinspect-cli-action PR
+
+- go to the PR [example](https://github.com/splunk/appinspect-cli-action/pull/127)
+    - check [release notes](https://dev.splunk.com/enterprise/docs/relnotes/relnotes-appinspectcli/whatsnew/) for the new version
```

Comment:
may be add release notes link in the PR comment?

Reply:
added